### PR TITLE
Induced scheduler

### PR DESCRIPTION
**Pull request by sjunges**
When applying a fully defined scheduler to an MDP, we obtain a DTMC; this allows applying DTMC-specific code on this model.

### CODE REVIEW

**Comment by sjunges** [comment]
Additionally added a flag that enforces that an MDP remains an MDP etc; as this is sometimes useful when writing concise code

File: src/storm/transformer/SubsystemBuilder.cpp [hunk 1]
```diff
@@ -134,6 +134,10 @@ SubsystemBuilderReturnType<ValueType, RewardModelType> internalBuildSubsystem(st
     } else {
         components.transitionMatrix = originalModel.getTransitionMatrix().getSubmatrix(false, keptActions, subsystemStates);
     }
+    if (options.makeRowGroupingTrivial) {
```
Comment (tquatmann) [inline src/storm/transformer/SubsystemBuilder.cpp:137]: Do we have to catch the case where the transitionMatrix is still supposed to be nondeterministic (i.e. not square)?

**Review by tquatmann — COMMENTED** [review]
I think it would be good to catch that one edge case in the subsystem builder with e.g. an assertion. Other then that, this looks good to me 👍

File: src/storm/transformer/SubsystemBuilder.cpp (hunk 1 above)
Comment (sjunges) [reply src/storm/transformer/SubsystemBuilder.cpp:137]: Yes, indeed, will do!

**Comment by sjunges** [comment]
This also relates to issue #69 (it doesnt solve it, but it alleviates some cases and now allows to use the subsystem builder appropriately).

**Pull request merged**